cloud_functions/token_refresh/utils/cloud_utils.py

```
@@ -0,0 +1,29 @@
+from os import environ
+
+from beartype import beartype
+from google.cloud import secretmanager
+from utils.logger_utils import logger
+
+
+@beartype
+def get_secret(name: str) -> dict:
+    """Retrieve secret from secret manager."""
+    if "PROJECT_ID" not in environ:
+        raise KeyError("PROJECT_ID environment variable is not set")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)